tests/runner.py

```diff
@@ -1622,11 +1622,11 @@ def build_library(name,
         with open(os.path.join(project_dir, 'configure_err'), 'w') as err:
           stdout = out if EM_BUILD_VERBOSE < 2 else None
           stderr = err if EM_BUILD_VERBOSE < 1 else None
-          building.configure(configure + configure_args, env=env,
+          building.configure(configure, env=env,
                              stdout=stdout,
                              stderr=stderr,
                              cwd=project_dir)
-    except subprocess.CalledProcessError:
+    except Exception:
```

Comment:
Why the change?

Reply:
I ill revert that this part since I'm planning a followup that will change this anyway.